doc/source/changelog.rst

```diff
@@ -17,7 +17,8 @@ What's new?
 
 * **Minor updates**
     * Update num_examples caclulation in PyTorch code examples in (`#909 <https://github.com/adap/flower/pull/909>`_)
-    * Expose Flower version through "flwr.__version__" (`#952 <https://github.com/adap/flower/pull/952>`_)
+    * Expose Flower version through "flwr.__version__" (`#952 <https://github.com/adap/flower/pull/952>`_)#974
```

Comment:
Why is there a #974 at the end of this line?

Reply:
Good catch @sisco0 , @pedropgusmao could you remove it?

Reply:
will change, thanks!